lib/ex_unit/test/ex_unit/doc_test_test.exs

```diff
@@ -206,64 +215,65 @@ defmodule ExUnit.DocTestTest do
 
     ExUnit.configure(seed: 0, colors: [enabled: false])
     output = capture_io(fn -> ExUnit.run end)
+             |> String.replace(~r/exs:\d+/, "exs:x") # Ignore line numbers
```

Comment:
I've removed the line number replace from the PR and updated the literal line numbers in the tests.